lib/git_reflow/commands/setup.rb

```diff
@@ -15,11 +15,15 @@
     end
 
     choose do |menu|
-      menu.header = "Available remote Git Server services:"
+      menu.header = "Available remote Git Server services"
       menu.prompt = "Which service would you like to use for this project?  "
 
       menu.choice('GitHub')    { GitReflow::GitServer.connect reflow_options.merge({ provider: 'GitHub', silent: false }) }
       menu.choice('BitBucket (team-owned repos only)') { GitReflow::GitServer.connect reflow_options.merge({ provider: 'BitBucket', silent: false }) }
     end
+
+    GitReflow::Config.add "constants.minimum_approvals", ask("Set the minimum number of approvals (leaving blank will require approval from all commenters): ")
```

Comment:
Just realized these should pass the `local` option in case the user want's to configure a project-specific git-reflow.  I think adding `local: reflow_options[:project_only]` will do it.